src/Dialogs/AdjustDateTimeDialog.vala

```diff
@@ -47,26 +47,29 @@ public class AdjustDateTimeDialog : Gtk.Dialog {
         24HR;
     }
 
-    TimeSystem previous_time_system;
+    private TimeSystem previous_time_system;
 
     construct {
         ui_settings = new GLib.Settings (GSettingsConfigurationEngine.UI_PREFS_SCHEMA_NAME);
         file_settings = new GLib.Settings (GSettingsConfigurationEngine.FILES_PREFS_SCHEMA_NAME);
+
+        add_buttons (
+            (_("_Cancel")), Gtk.ResponseType.CANCEL,
+            (_("_Apply")), Gtk.ResponseType.OK
+        );
+
+        deletable = false;
+        modal = true;
+        resizable = false;
+        title = _(Resources.ADJUST_DATE_TIME_LABEL);
+        transient_for = AppWindow.get_instance ();
+        set_default_response (Gtk.ResponseType.OK);
     }
```

Comment:
Could set properties in an Object () statement in the constructor. Not sure which is preferred.